models/bg.model.js

```diff
@@ -8,4 +8,14 @@ exports.fetchCategories = () => {
         .then((categories) => {
             return categories.rows
         })
+}
+
+exports.fetchReviews = () => {
+    return db
+        .query(
+            `select reviews.*, comments.review_id as comment_count from reviews join comments on reviews.review_id = comments.review_id order by reviews.created_at desc`
```

Comment:
comment_count is supposed to be the total count of all the comments that correspond with this review_id - you are currently not counting anything, have a look at the COUNT function.

Furthermore, you're using "join" which is essentially an "INNER JOIN". An inner join returns rows when there is a match in both tables. 
This is not the correct choice for the task at hand - have another think about which join we'd want to use here!